pyproject.toml

```
@@ -25,16 +25,15 @@ classifiers = [
     "Programming Language :: Python",
     "Programming Language :: Python :: 3",
     "Programming Language :: Python :: 3 :: Only",
-    "Programming Language :: Python :: 3.7",
     "Programming Language :: Python :: 3.8",
     "Programming Language :: Python :: 3.9",
     "Programming Language :: Python :: 3.10",
     "Programming Language :: Python :: 3.11",
     "Programming Language :: Python :: 3.12",
 ]
-requires-python = ">=3.7"
+requires-python = ">=3.8"
 dependencies = [
-    "execnet>=1.1",
+    "execnet@git+https://github.com/pytest-dev/execnet#egg=master",
```

Once a new execnet is released:

```suggestion
    "execnet>=2.1.0",
```